models/src/test/java/io/sphere/sdk/reviews/ReviewDraftTest.java

```diff
@@ -0,0 +1,142 @@
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by njacinto on 21/04/15.
+ */
+public class ReviewDraftTest {
+    static final ReviewDraft DEFAULT_REVIEW;
+
+
+    static {
+        DEFAULT_REVIEW = ReviewDraft.of(ReviewImplTest.DEFAULT_PRODUCT_ID, ReviewImplTest.DEFAULT_CUSTOMER_ID)
+                .withAuthorName(ReviewImplTest.DEFAULT_AUTHOR_NAME).withTitle(ReviewImplTest.DEFAULT_TITLE)
+                .withText(ReviewImplTest.DEFAULT_TEXT).withScore(ReviewImplTest.DEFAULT_SCORE);
+    }
+
+    @Test
+    public void testOf() throws Exception {
+        ReviewDraft review = ReviewDraft.of(ReviewImplTest.DEFAULT_PRODUCT_ID, ReviewImplTest.DEFAULT_CUSTOMER_ID);
+        assertEquals(ReviewImplTest.DEFAULT_PRODUCT_ID, review.getProductId());
+        assertEquals(ReviewImplTest.DEFAULT_CUSTOMER_ID, review.getCustomerId());
+    }
+
+    @Test
+    public void testGetProductId() throws Exception {
+        assertEquals(ReviewImplTest.DEFAULT_PRODUCT_ID, DEFAULT_REVIEW.getProductId());
+    }
+
+    @Test
+    public void testGetCustomerId() throws Exception {
+        assertEquals(ReviewImplTest.DEFAULT_CUSTOMER_ID, DEFAULT_REVIEW.getCustomerId());
+    }
+
+    @Test
+    public void testGetAuthorName() throws Exception {
+        assertEquals(ReviewImplTest.DEFAULT_AUTHOR_NAME, DEFAULT_REVIEW.getAuthorName().get());
+    }
+
+    @Test
+    public void testGetTitle() throws Exception {
+        assertEquals(ReviewImplTest.DEFAULT_TITLE, DEFAULT_REVIEW.getTitle().get());
+    }
+
+    @Test
+    public void testGetText() throws Exception {
+        assertEquals(ReviewImplTest.DEFAULT_TEXT, DEFAULT_REVIEW.getText().get());
+    }
+
+    @Test
+    public void testGetScore() throws Exception {
+        assertTrue(ReviewImplTest.isScoreInValidRange(DEFAULT_REVIEW.getScore().get(), ReviewImplTest.DEFAULT_SCORE));
+    }
+
+    @Test
+    public void testWithTitle() throws Exception {
+        ReviewDraft review = ReviewDraft.of(ReviewImplTest.DEFAULT_PRODUCT_ID, ReviewImplTest.DEFAULT_CUSTOMER_ID);
+        assertFalse(review.getTitle().isPresent());
+        ReviewDraft reviewWith = review.withTitle(ReviewImplTest.DEFAULT_TITLE);
+        assertFalse(review==reviewWith);
+        assertEquals(ReviewImplTest.DEFAULT_TITLE, reviewWith.getTitle().get());
+    }
+
+    @Test
+    public void testWithText() throws Exception {
+        ReviewDraft review = ReviewDraft.of(ReviewImplTest.DEFAULT_PRODUCT_ID, ReviewImplTest.DEFAULT_CUSTOMER_ID);
+        assertFalse(review.getText().isPresent());
+        ReviewDraft reviewWith = review.withText(ReviewImplTest.DEFAULT_TEXT);
+        assertFalse(review==reviewWith);
+        assertEquals(ReviewImplTest.DEFAULT_TEXT, reviewWith.getText().get());
+    }
+
+    @Test
+    public void testWithScore() throws Exception {
+        ReviewDraft review = ReviewDraft.of(ReviewImplTest.DEFAULT_PRODUCT_ID, ReviewImplTest.DEFAULT_CUSTOMER_ID);
+        assertFalse(review.getScore().isPresent());
```

Comment:
I would skip this assertion since you are getting the value later, what do you think?